test-cases/longevity/longevity-1TB-7days-authorization-and-tls-ssl-1dis-2nondis-nemesis.yaml

```diff
@@ -32,8 +32,7 @@ user_prefix: 'longevity-tls-1tb-7d-1dis-2nondis'
 failure_post_behavior: destroy
 space_node_threshold: 644245094
 server_encrypt: 'true'
-# Setting client encryption to false for now, till we will find the way to make c-s work with that
-client_encrypt: 'false'
+client_encrypt: True
```

Comment:
for consistency let's use: `client_encrypt: true` instead "True" (with uppercase)
also no need to put single quotation marks around the "true" 
should be `server_encrypt: true`

Reply:
Fixed.